com.unity.render-pipelines.universal/Editor/Converter/Converters/RenderSettingsConverter.cs

```diff
@@ -0,0 +1,301 @@
+            m_GraphicsTierSettings.ReflectionProbeBlending = tier.reflectionProbeBlending;
+            m_GraphicsTierSettings.ReflectionProbeBoxProjection = tier.reflectionProbeBoxProjection;
+            m_GraphicsTierSettings.CascadeShadows = tier.cascadedShadowMaps;
+            m_GraphicsTierSettings.HDR = tier.hdr;
+        }
+
+        /// <summary>
+        /// Iterates over all Quality Settings and saves relevant settings to a RenderSettingsItem.
+        /// This will also create the required information for the Render Pipeline Converter UI.
+        /// </summary>
+        /// <param name="context">Converter context to add elements to.</param>
+        private void GatherQualityLevels(ref InitializeConverterContext context)
+        {
+            var currentQuality = QualitySettings.GetQualityLevel();
+            var id = 0;
+            foreach (var levelName in QualitySettings.names)
+            {
+                QualitySettings.SetQualityLevel(id);
+
+                var projectSettings = new RenderSettingItem
+                {
+                    Index = id,
+                    LevelName = levelName,
+                    PixelLightCount = QualitySettings.pixelLightCount,
+                    MSAA = QualitySettings.antiAliasing,
+                    Shadows = QualitySettings.shadows,
+                    ShadowResolution = QualitySettings.shadowResolution,
+                    ShadowDistance = QualitySettings.shadowDistance,
+                    ShadowCascadeCount = QualitySettings.shadowCascades,
+                    CascadeSplit2 = QualitySettings.shadowCascade2Split,
+                    CascadeSplit4 = QualitySettings.shadowCascade4Split,
+                    SoftParticles = QualitySettings.softParticles,
+                };
+                m_SettingsItems.Add(projectSettings);
+
+                var setting = QualitySettings.GetRenderPipelineAssetAt(id);
+                var item = new ConverterItemDescriptor {name = $"Quality Level {id}: {levelName}"};
+
+                if (setting != null)
+                {
+                    item.warningMessage = setting.GetType() == typeof(UniversalRenderPipelineAsset)
+                        ? "Contains URP Asset, will override existing asset."
+                        : "Contains SRP Asset, will override existing asset with URP asset.";
+                }
+
+                context.AddAssetToConvert(item);
+                id++;
+            }
+
+            QualitySettings.SetQualityLevel(currentQuality);
+        }
+
+        public override void OnRun(ref RunItemContext context)
+        {
+            var item = context.item;
+            // is quality item
+            if (m_SettingsItems[item.index].GetType() == typeof(RenderSettingItem))
+            {
+                GeneratePipelineAsset(item.index, m_SettingsItems[item.index] as RenderSettingItem);
+            }
+        }
+
+        private void GeneratePipelineAsset(int index, RenderSettingItem settings)
```

Comment:
```suggestion
        private void GeneratePipelineAsset(RenderSettingItem settings)
```
This is not being used.